polaris-tokens/scripts/toTokenValues.ts

```diff
@@ -1,32 +1,20 @@
 import fs from 'fs';
 import path from 'path';
 
-import type {MetaTokens, Tokens} from '../src';
+import {removeMetadata} from '../src';
+import type {Entry, Entries, MetaTokens, Tokens} from '../src/types';
 
 const outputDir = path.join(__dirname, '../build');
 const outputFile = path.join(outputDir, 'index.ts');
 
-type Entry<T> = [keyof T, T[keyof T]];
-type Entries<T> = Entry<T>[];
-
 export async function toTokenValues(metaTokens: MetaTokens) {
   if (!fs.existsSync(outputDir)) {
     await fs.promises.mkdir(outputDir);
   }
 
   const tokensEntries: Entries<Tokens> = Object.entries(metaTokens).map(
-    (entry) => {
-      const [tokenGroupName, tokenGroup] = entry as Entry<MetaTokens>;
-
-      const tokenGroupValues = Object.fromEntries(
-        Object.entries(tokenGroup).map(([tokenName, {value}]) => [
-          tokenName,
-          value,
-        ]),
-      );
-
-      return [tokenGroupName, tokenGroupValues];
-    },
+    ([tokenGroupName, tokenGroup]) =>
+      [tokenGroupName, removeMetadata(tokenGroup)] as Entry<Tokens>,
```

Comment:
```suggestion
    (entry): Entry<Tokens> => {
      const [tokenGroupName, tokenGroup] = entry as Entry<MetaTokens>;

      return [tokenGroupName, removeMetadata(tokenGroup)];
    },
```

Similar to the comment below, casting the `entry` itself allows us to validate the `.map` return type matches `Entry<Tokens>`.